src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -1887,6 +1887,47 @@ public void testDeleteDatasetWhileFileIngesting() {
                 .statusCode(FORBIDDEN.getStatusCode());
 
     }
+    
+    @Test
+    public void testGetIncludeOwnerArray() {
```

Comment:
```suggestion
    public void testGetDatasetOwners() {
```

Just to be consistent with the other test method names.